lstchain/mc/sensitivity.py

```diff
@@ -179,10 +179,17 @@ def calculate_sensitivity_lima(n_excesses, n_background, alpha, n_bins_energy, n
 
     """
 
+    if any(len(a) != n_bins_energy for a in (n_excesses, n_background, alpha)):
+        raise ValueError(
+            'Excess, background and alpha arrays must have the same length')
+           
+    
     stat = WStatCountsStatistic(
-        n_on=np.ones_like(n_background),
-        n_off=n_background,
-        alpha=alpha)
+        n_on = n_excesses,  
```

Comment:
here, if you want to calculate the sensitivity using this quantity, you would need to include not the excesses but the number of events in the on region (`n_excesses` + `n_background`). It is anyways not necessary, because we only use it to calculate the excess matching a 5 sigma significance, that is not dependent on `n_excesses`

Reply:
I agree. Actually I did not modify this part, but I think someone wrote "n_excess" but it is actually the number of ON events. In the following parts of the program where it is used, the number of ON events is provided.

I am changing the variables name in the function, for better readibility: from n_excess to n_on_events

Thank you